setup.cfg

```diff
@@ -10,10 +10,12 @@ branch = True
 source = src
 
 [pycodestyle]
+ignore = E302,E301,W504  # E302 & E301 due pycodestyle's awful support for @overload
+select = W503  # W503 looks better than W504
 max-line-length = 100
 
 [tool:pytest]
-testpaths = test
+testpaths = src test
```

Comment:
This new `src` will allow to check the style in all the modules, right?

Reply:
Yes, it recursively checks all `.py` files in https://github.com/SCM-NV/qmflows/tree/master/src.